### DISCUSSION TITLE
Wiggle mod and Autoplay do not work well together

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#20575] [Q&A] [answered]
i havent seen any other mention of this .  i suppose the reason why it hasnt been reported is not many people would actually want to touch wiggle mod high CS with a ten foot pole, somewhat understandably. 

Regardless it seem auto does not have the ability to zero in on moving notes ( as was duly noted for magnetised and repel) shall i push a mini PR to make them incompatible? leave it as is and work on some sort of solution ?..... 

https://user-images.githubusercontent.com/74463310/193834675-42b8d969-9bfe-4659-b485-010ba92d15d0.mov

**Comment by peppy** [comment, marked as answer]
At some point we may want to change how autoplay operates in general. To maintain compatibility with lower overhead, it might make more sense to have it run in realtime rather than pre-generate a replay (ie. aim for the next circle with more movement the close it is to the hit time, to ensure it can track moving targets etc.).

Needs further investigation.
